src/status_im/ui/screens/wallet/send/db.cljs

```diff
@@ -7,8 +7,9 @@
 (spec/def ::to (spec/nilable string?))
 (spec/def ::to-name (spec/nilable string?))
 (spec/def ::amount-error (spec/nilable string?))
+(spec/def ::asset-error (spec/nilable string?))
 (spec/def ::amount-text (spec/nilable string?))
-(spec/def ::password (spec/nilable string?))
+(spec/def ::password (spec/nilable (constantly true)))
```

Comment:
Is that change intended?

Reply:
Good catch, no. This was temporary workaround for the spec issue with masked password. Just a sec